runtime/doc/intro.txt

```diff
@@ -536,7 +536,7 @@ CTRL-O in Insert mode you get a beep but you are still in Insert mode, type
  Select		 *5	^O ^G		 *6	    --	      --      --
  Insert		 <Esc>	  --	  --		  <Insert>    --      --
  Replace	 <Esc>	  --	  --	<Insert>	      --      --
- Command-line	 `*3`	  --	  --	 :start	    --		      --
+ Command-line	 *3	  --	  --	 :start	    --		      --
```

Comment:
this was a workaround for the parser / HTML generation.

https://github.com/neovim/neovim/issues/30937 is related, though based on @zeertzjq 's comment there I infer that this might be a potential fix?